mmdet/evaluation/metrics/coco_metric.py

```diff
@@ -304,21 +304,21 @@ def gt_to_coco_json(self, gt_dicts: Sequence[dict],
         return converted_json_path
 
     def process(self, data_batch: Sequence[dict],
-                predictions: Sequence[dict]) -> None:
+                data_samples: Sequence[dict]) -> None:
         """Process one batch of data samples and predictions. The processed
         results should be stored in ``self.results``, which will be used to
         compute the metrics when all batches have been processed.
 
         Args:
             data_batch (Sequence[dict]): A batch of data
                 from the dataloader.
-            predictions (Sequence[dict]): A batch of outputs from
-                the model.
+            data_samples (Sequence[dict]): A batch of data samples that
+                contain annotations and predictions.
         """
-        for data, pred in zip(data_batch, predictions):
+        for data, data_sample in zip(data_batch, data_samples):
```

Comment:
Should not transverse `data_batch` here, since `data_batch` is a dict.

Reply:
OK